src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.cc

```diff
@@ -1767,6 +1767,10 @@ void GrpcLb::CreateOrUpdateChildPolicyLocked() {
     update_args.addresses = serverlist_->GetServerAddressList(
         lb_calld_ == nullptr ? nullptr : lb_calld_->client_stats());
     is_backend_from_grpclb_load_balancer = true;
+    if (update_args.addresses.ok() && update_args.addresses->empty()) {
+      update_args.resolution_note =
+          absl::StrCat("empty list from balancer: ", resolution_note_);
```

Comment:
There's no point in using `resolution_note_` here, because it won't contain anything relevant.  `resolution_note_` is set based on the resolution note passed down from the resolver.  But in this case, we're using an address list from the balancer, not from the resolver.

This should just be:

```
update_args.resolution_note = "empty serverlist from grpclb balancer";
```

Reply:
Done.